demo/kserve/scripts/install/kserve-install.sh

```diff
@@ -9,6 +9,13 @@ set -o errtrace
 
 source "$(dirname "$(realpath "$0")")/../env.sh"
 source "$(dirname "$(realpath "$0")")/../utils.sh"
+if [[ ! -n ${CHECK_UWM+x} ]]
```

Comment:
Do we need this logic to get input from inline every time? This is optional check logic so I don't think it is required.

Reply:
I think it's better to ask for it to be consistent with the other variables
> Note: If you do not set values for these variables before you run the script, the script asks for them.


from the [README](https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/scripts/README.md)